flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/metrics/OperatorJosdkMetricsTest.java

```diff
@@ -45,15 +48,18 @@
 /** {@link OperatorJosdkMetrics} tests. */
 public class OperatorJosdkMetricsTest {
 
-    private final ResourceID resourceId = new ResourceID("testname", "testns");
-    private final String controllerName = FlinkDeploymentController.class.getSimpleName();
-    private final String resourcePrefix =
-            "testhost.k8soperator.flink-operator-test.testopname.resource.testns.testname.JOSDK.";
-    private final String systemPrefix =
+    private static final ResourceID resourceId = new ResourceID("testname", "testns");
+    private static final String controllerName = FlinkDeploymentController.class.getSimpleName();
+    private static final String resourcePrefix =
```

Comment:
We can consider using the testing metric listener here too to avoid backing in the scope names.

Reply:
I will try to improve this, good point